Srcs/heiftojpeg/heiftojpeg.cpp

```diff
@@ -0,0 +1,183 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include "hevcimagefilereader.hpp"
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <Magick++.h>
+#include <list>
+
+using namespace std;
+
+static void decodeData(ImageFileReaderInterface::DataVector data, Magick::Image *image)
+{
+    std::string hevcFileName = "tmp.hevc";
+    std::string bmpFileName = "tmp.bmp";
+    std::ofstream hevcFile(hevcFileName);
+    hevcFile.write((char*)&data[0],data.size());
+    hevcFile.close();
+    system(("ffmpeg -i tmp.hevc -loglevel panic -frames:v 1 -vsync vfr -q:v 1 -y -an " + bmpFileName).c_str());
```

Comment:
This should reuse `hevcFileName` instead of hard code "tmp.hevc". I worry that this uses the same file name which could cause problems if more than one copy is run in the same container at the same time. Also: the `system` return code is ignored. The same is true of `write` failures.

Reply:
done. i also added error checking for system calls and file writing.